xcode/Subconscious/Shared/Models/EntryLink.swift

```diff
@@ -36,18 +36,6 @@ struct EntryLink:
         )
     }
     
-    /// Construct an EntryLink from a string.
```

Comment:
This initializer gets moved to MemoAddress.swift. Constructing an EntryLink from slug and audience requires MemoAddress-specific switch logic, so we want to keep it with MemoAddress, so we can easily update it if the MemoAddress type changes in the future.